src/vivarium_census_prl_synth_pop/results_processing/formatter.py

```diff
@@ -41,7 +46,9 @@ def get_state_name(data: pd.DataFrame) -> pd.Series:
     "first_name_id": (get_first_name_id, ["first_name_id", "random_seed"]),
     "middle_name_id": (get_middle_name_id, ["middle_name_id", "random_seed"]),
     "last_name_id": (get_last_name_id, ["last_name_id", "random_seed"]),
-    "state": (get_state_name, ["state"]),
+    # This is a temp fix until the pipeline is change in the simulation and needs to happen before abbreviation
```

Comment:
Is there a ticket we can reference here?

Reply:
I think it is the ticket Steve is working on and this should only be here for a few days.  Ill add that.